src/Input/Input.component.tsx

```diff
@@ -171,6 +167,11 @@ const SInputWrapper = styled.div`
       margin-left: -1px;
     }
   }
+  &.disabled, &.disabled > input {
```

Comment:
Seems like setting the `invalid` prop to true does not add the invalidBorderColor

![image](https://user-images.githubusercontent.com/20873103/58332338-b240a380-7e43-11e9-8ba2-0592eed7a206.png)